bucket/bucket.go

```diff
@@ -50,6 +51,20 @@ func (b *Bucket) GetObjectReader(key string, opts ...option.GetObjectInput) (io.
 	return resp.Body, nil
 }
 
+// GetObjectRequest generates a "aws/request.Request" representing the client's request for the GetObject operation.
```

Comment:
so now `*Bucket.GetObject` could be the following as [the SDK does](https://github.com/aws/aws-sdk-go/blob/master/service/s3/api.go#L2843):

```go
func (b *Bucket) GetObject(key string, opts ...) (*s3.GetObjectOutput, error) {
    	req, out := b.GetObjectRequest(key, opts...)
	return out, req.Send()
}
```